Tutorial_1c_Line_Ratios.ipynb

```diff
@@ -0,0 +1,736 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #62.    structure = np.dstack([structure_spec, structure, structure_spec])
</pre><p>update to <code>np.swapaxes(np.dstack([structure_spec, structure, structure_spec]), 0, 2)</code> to fix the bug I mentioned for the other tutorial.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Smithsonian/sma-science-tutorials/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>